File: droid-app/CoffeeDose/app/src/main/java/com/office14/coffeedose/domain/Order.kt
package com.office14.coffeedose.domain

data class Order (
    var id : Int,
    val statusCode : String,
    val statusName : String,
    val orderNumber : String,
    val totalPrice : Int,
    val owner : String?,
    val isFinished : String
)

enum class OrderStatus {
    NONE, COOKING, READY, FAILED;

    companion object{
        fun getStatusByString(status: String) : OrderStatus{
            return when(status.toLowerCase()){
                "cooking" -> OrderStatus.COOKING
                "ready" -> OrderStatus.READY
                else -> OrderStatus.NONE
            }
        }
    }


}

File: web-client/src/app/order/order.ts
export interface Order {
	"id": number,
    "status_code": string,
    "status_name": string,
    "order_number": string,
    "total_price": number,
    "comment": string,
	"drinks": Array<object>
    "became_ready"?: boolean,
    "created"?: Date
}